public/script.js

```diff
@@ -77,15 +73,43 @@ async function animationLoop(){
                 TIME: STATE.current.format("YYYY-MM-DD"),
                 STYLES: "NSIDC:g02135_" + STATE.extCon + "_raster_basemap"
             };
+            STATE.rate = 2000 - $("#speedSlider").val();
             await updateWMSLayerParams(map.getLayers().getArray()[0],wmsParams);
-            await sleep(STATE.rate);
+            await sleep(STATE.rate);      
         }
         else {
             await sleep(50);
         }
     }
 }
 
+function getState(map, projection) {
+    var oldHemisphere = STATE.norSouth;
+    STATE.extCon = $('input[name=ext-con]:checked').val();//Get value for extent or concentration
+    STATE.norSouth = $('input[name=n-s]:checked').val();//Get value for North or South
+    STATE.dateLoopStyle = $('input[name=dates]:checked').val();//Get value for the looping style
+    STATE.monthLoop = $('select[name=monthsLoop]').val();//Month to be repeated if that option is chosen
+    STATE.start = moment(document.querySelector('input[name="sDate"]').value)
+    STATE.end = moment(document.querySelector('input[name="eDate"]').value)
+    if (oldHemisphere != STATE.norSouth) {
+        
+        var wmsParams = {
+            LAYERS: "NSIDC:g02135_" + STATE.extCon + "_raster_daily_" + STATE.norSouth,
+            SRS: getLocationParams().srs,
+            BBOX: getLocationParams().locationVal,
+            TILED: false,
+            format:"image/png",
+            TIME: STATE.current.format("YYYY-MM-DD"),
+            STYLES: "NSIDC:g02135_" + STATE.extCon + "_raster_basemap"
+        };
+        $("#map").html("");//Empty map when a new animation occurs
```

Comment:
Does Openlayers have any methods to cleanup the map, or is this really all that needs to be done?